vm/loader/page_table/src/x64.rs

```diff
@@ -299,289 +392,311 @@ impl PteOps for PageTableBuilder {
     }
 }
 
-impl PageTableBuilder {
-    pub fn new(page_table_gpa: u64) -> Self {
-        PageTableBuilder {
-            page_table_gpa,
-            start_gpa: 0,
-            size: 0,
-            local_map: None,
-            confidential_bit: None,
-            map_reset_vector: false,
-        }
-    }
+impl<'a> PageTableBuilder<'a> {
+    /// Creates a new instance of PageTableBuilder, taking required arguments for working buffers and the page table gpa.
```

Comment:
this comment isn't very helpful - could you be more specific on what each argument means?